steps/preprocessing.py

```diff
@@ -3,12 +3,20 @@
 
 import numpy as np
 import pandas as pd
+
 from sklearn.externals import joblib
 from sklearn.feature_extraction import text
 import sklearn.preprocessing as sk_prep
-
+from nltk.tokenize import word_tokenize
```

Comment:
@shaz13 ,
This introduces additional dependency, that is `nltk` -> this should be added to the requirements.txt

Reply:
Done.

Reply:
@shaz13 Nice thanks

Reply:
Done